.github/workflows/validate-semantic-pr.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Validate Pull Request
-        uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb
+        uses: amannn/action-semantic-pull-request@0b14f54ac155d88e12522156e52cb6e397745cfd
```

Comment:
Maybe we could just use v5 here?

Reply:
I have a PR for that already https://github.com/keptn/lifecycle-toolkit/pull/360